commands/select.md

```diff
@@ -5,6 +5,6 @@ Selectable Valkey databases are a form of namespacing: all databases are still p
 
 In practical terms, Valkey databases should be used to separate different keys belonging to the same application (if needed), and not to use a single Valkey instance for multiple unrelated applications.
 
-When using Valkey Cluster, only `SELECT 0` can be used, since Valkey Cluster only supports database zero. In the case of a Valkey Cluster, having multiple databases would be useless and an unnecessary source of complexity. Commands operating atomically on a single database would not be possible with the Valkey Cluster design and goals.
+Valkey Cluster now allows multiple databases. You can use `SELECT <dbid>`  to switch between them.
```

Comment:
Avoid "now". Users run different versions and they need to know version. Also, docs can become outdated.

Ideas:

* Valkey 9.0 and later allows multiple...
* Since Valkey 9.0, ...
* Starting with Valkey 9.0, ...

Reply:
or "since version", "starting with version", etc.